assets/css/default.css

```diff
@@ -361,7 +361,7 @@ footer #copyright-text {
 }
 
 .trickfire-green {
-  background-color: #38d738;
+  background-color: #52d152;
```

Comment:
change this to $primary-color

Reply:
That is the css file, not the scss file. The default.scss does have the $primary-color. 
With the new changes, the trickfire-green is actually going to stay (56,215,56) instead of some other color.

Reply:
Thank you. I'm just dumb